### PR TITLE
meter regwall

### PR DESCRIPTION
**Pull request by chenshay**

### CODE REVIEW

File: src/runtime/meter-regwall-flow.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    /** @private {?Promise} */
+    this.openViewPromise_ = null;
+
+    /** @private @const {!ActivityIframeView} */
+    this.activityIframeView_ = new ActivityIframeView(
+      this.win_,
+      this.activityPorts_,
+      feUrl('/meterregwalliframe'),
+      feArgs({
```
Comment (jpettitt) [inline src/runtime/meter-regwall-flow.js:47]: will need to  also pass in the iframe url for the GSI sign in helper shim and a url to render for the already registered link.

Comment (chenshay) [reply src/runtime/meter-regwall-flow.js]: done

**Review by ChrisAntaki — APPROVED** [review]

File: src/runtime/meter-regwall-api.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    /** @private @const {!../components/dialog-manager.DialogManager} */
+    this.dialogManager_ = deps.dialogManager();
+
+    /** @private @const {!ActivityIframeView} */
+    this.activityIframeView_ = new ActivityIframeView(
+      this.win_,
+      this.activityPorts_,
+      feUrl('/meterregwalliframe'),
+      feArgs({
```
Comment (mborof) [inline src/runtime/meter-regwall-api.js:48]: deps.activities().addDefaultArguments({
        gsiHelperIframe: meterRegwallArgs.gsiHelperIframe,
        alreadyRegisteredUrl: meterRegwallArgs.alreadyRegisteredUrl,
});

https://github.com/subscriptions-project/swg-js/blob/main/src/components/activities.js#L275

**Comment by ChrisAntaki** [comment]
Thanks for your formative work on the Meter Regwall, @chenshay !

**Pull request closed without merging**